lib/createHandler.ts

```diff
@@ -10,15 +10,12 @@ import { notFound } from './internals/notFound';
  *
  * @example
  * ```ts
- * import { createHandler, Get } from '[at]storyofams/next-api-decorators';
+ * import { createHandler, Get } from '@storyofams/next-api-decorators';
  *
  * class Events {
  *  Get()
```

Comment:
Didn't find a way to escape the `@` sign in the code block for `@Get` that displayed correctly in VSC..